src/common/main/kotlin/tech/thatgravyboat/skyblockapi/api/events/base/predicates/InventoryPredicates.kt

```diff
@@ -19,10 +19,15 @@ annotation class InventoryTitle(
 @Target(AnnotationTarget.FUNCTION)
 annotation class MustBeContainer
 
+@Retention(AnnotationRetention.RUNTIME)
+@Target(AnnotationTarget.FUNCTION)
+annotation class IgnoreFiller
+
 class InventoryPredicates : EventPredicateProvider {
     override fun getPredicate(method: Method): EventPredicate? {
         val inventoryTitle = method.getAnnotation<InventoryTitle>()?.title?.map { Regex(it) }
         val disallowPlayerInventory = method.getAnnotation<MustBeContainer>() != null
+        val ignoreFiller = method.getAnnotation<IgnoreFiller>() != null
```

Comment:
this wont apply if the other annotations arnt also applied, this should be a separate predicate that is created if the annotation is on the method.

Reply:
We do this exact same thing for every single predicate so far?

Reply:
no like right below this is does `if (inventoryTitle == null && !disallowPlayerInventory) return` meaning if it doesnt have the other 2 annotations it wont work :pleading_face: